src/lib.rs

```diff
@@ -21,9 +22,11 @@ pub use sqlx_core::{query, query_as};
 pub use sqlx_core::query_as_mapped;
 
 #[cfg(feature = "mysql")]
+#[cfg_attr(docsrs, doc(cfg(feature = "mysql")))]
 pub use sqlx_core::mysql::{self, MySql, MySqlConnection, MySqlPool};
 
 #[cfg(feature = "postgres")]
+#[cfg_attr(docsrs, doc(cfg(feature = "postgres")))]
 pub use sqlx_core::postgres::{self, PgConnection, PgPool, Postgres};
 
 #[cfg(feature = "macros")]
```

Comment:
Could you cover the `macros` module as well, please? I'd just do the `macro_rules!` items inside.